simd/src/Kokkos_SIMD_AVX2.hpp

```diff
@@ -680,6 +680,8 @@ template <>
 class simd<std::int64_t, simd_abi::avx2_fixed_size<4>> {
   __m256i m_value;
 
+  static_assert(sizeof(long long) == 8);
```

Comment:
Why did you add that assertion?

Reply:
As a minimal safeguard that the `reinterpret_cast` is reasonable.